source/manual/how-to-be-a-good-buddy.html.md

```diff
@@ -0,0 +1,38 @@
+---
+owner_slack: "#govuk-developers"
+title: How to be a good buddy
+section: Learning GOV.UK
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+---
+
+When a new joiner starts on govuk, it's good practice to assign them a "buddy" to help them get started on govuk and it's infrastructure and provide initial support.
+
+## What is a buddy?
+
+A buddy acts as a temporary guide for new joiners, helping them understand the govuk programme and generally get started. Some responsibilities of a buddy can include:
+
+- explaining how our stack works
+- pointing the new joiner to useful docs/slack channels/general resources
+- being on hand to answer specific questions
+- generally make them feel welcome
+
+Ideally, a buddy will be in the same team and sit within the same function as the new joiner eg: a new frontend developer would have a frontend buddy.
+
+## How does a buddy differ from a line manager?
+
+Line managers aren't the same as buddies in that they…
+
+- might not always be on the same team or even the same programme
+- are focused more on personal stuff like objectives and working patterns
+- have enough to deal with navigating SOP
+
+## Advice on being an effective buddy
+
+- Make sure that our new joiner documentation is up to date before they start
+- Schedule catchups with them when they start to talk through things like our stack and our tooling
+- Check in semi-regularly to make it clear that you're available if they need help with anything
+- Let your team know that you're acting as a new joiner's buddy so your capacity might be reduced
+- Consider having a handover discussion after the first month to check how the new joiner found on boarding and feed that feedback into our general onboarding process
+- Consider maintaining the buddy relationship after they've completed their onboarding to help the new joiner continue learning
```

Comment:
This advice feels a bit unofficial to be included in this otherwise official-looking list. I'd consider closing with something like this instead:

```suggestion

There's no official end date for being a buddy. It is up to you and the new joiner to continue checking in for as long as you both feel it's useful.
```